Artifacts/linux-eris-blockchain/setup.sh

```diff
@@ -0,0 +1,167 @@
+
+# -----------------------------------------------------------------------------
+# LICENSE
+
+# The MIT License (MIT)
+# Copyright (c) 2016-Present Eris Industries, Ltd.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+# -----------------------------------------------------------------------------
+# REQUIREMENTS
+
+# Ubuntu
+# Docker (**unless** INSTALL_DOCKER is not blank)
+
+# -----------------------------------------------------------------------------
+# USAGE
+
+# setup.sh USER [SERVICESTOSTART] [CHAINSTOSTART]
+
+# -----------------------------------------------------------------------------
+# Set defaults
+
+erisUser=$1
+services=( $(echo $2 | tr "," "\n") )
+chains=( $(echo $3 | tr "," "\n") )
+toStart=( "${services[@]}" "${chains[@]}" )
+
+# -----------------------------------------------------------------------------
+# Defaults
+
+GOVERSION="1.5"
+NODEVERSION="4"
+
+# -----------------------------------------------------------------------------
+# Install dependencies
+
+echo "Hello there! I'm the marmot that installs Eris."
+echo
+echo
+echo "Grabbing necessary dependencies"
+export DEBIAN_FRONTEND=noninteractive
+curl -sSL https://deb.nodesource.com/setup_"$NODEVERSION".x | sudo -E bash - &>/dev/null
+sudo apt-get install -y jq gcc git build-essential nodejs &>/dev/null
+curl -sSL https://storage.googleapis.com/golang/go"$GOVERSION".linux-amd64.tar.gz | sudo tar -C /usr/local -xzf - &>/dev/null
+if [ -n "$INSTALL_DOCKER" ]
+then
+  curl -sSL https://get.docker.com/ | sudo -E bash - &>/dev/null
+fi
+sudo usermod -a -G docker $erisUser &>/dev/null
+echo "Dependencies Installed."
+echo
+echo
+
+# -----------------------------------------------------------------------------
+# Getting chains
+
+echo "Getting Chain managers"
+curl -sSL -o /home/$erisUser/simplechain.sh https://raw.githubusercontent.com/Azure/azure-devtestlab/master/Artifacts/linux-eris-blockchain/simplechain.sh
```

Comment:
The other script will be deployed to the same location as this scrip and should be referenced locally.